app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -438,10 +438,14 @@ function AcuantCapture(
     const { image, cardtype, dpi, moire, glare, sharpness } = nextCapture;
     const isAssessedAsGlare = glare < glareThreshold;
     const isAssessedAsBlurry = sharpness < sharpnessThreshold;
+    const isUnsupportedDocument = cardtype !== 1;
```

Comment:
Is there any other possible card type, if so, may be we can have some constant for it.

Reply:
There is this [already defined type](https://github.com/18F/identity-idp/blob/main/app/javascript/packages/document-capture/components/acuant-camera.tsx#L96-L103) of the `AcuantDocumentType`.  Is that what you had in mind?

Reply:
@eileen-nava yes.